src/static-layout.js

```diff
@@ -173,6 +173,108 @@ export function applyStaticLayout(element, fixIeIntrinsic = false) {
   // If the layout was already done by server-side rendering (SSR), then the
   // code below will not run. Any changes below will necessitate a change to SSR
   // and must be coordinated with caches that implement SSR. See bit.ly/amp-ssr.
+  const {height, layout, width} = getEffectiveLayout(element);
```

Comment:
Does any of this code change as it is moved?

Reply:
The only change was that I return the these three values instead of them all being defined inline